### DISCUSSION TITLE
Microsoft Intune and the Curious Case of the Converting Firewall Rule Policy - odds+endpoints

### DISCUSSION THREAD

**Discussion by giscus** [ennnbeee/oddsandendpoints-web#7] [Comments]
# Microsoft Intune and the Curious Case of the Converting Firewall Rule Policy

Microsoft Intune mdm firewall rule policies in Endpoint Security automatically converting to the new Settings Catalog profiles using the Migration Tool.

https://memv.ennbee.uk/posts/firewall-rule-policy-conversion/

**Comment by Ginolard** [comment]
Fantastic!  I have been experimenting today with having MDE settings managed by Intune for servers.  I wasn't looking forward to having to manually create new FW rules policies for the MicrosoftSense format.  This blog post saved me a whole heap of trouble.  I just duplicated the old policy and waited 10 minutes and the magic happened in the background.

**Reply by ennnbeee** [reply]
Glad that this post helped. I was very confused as to the behaviour of the rule conversion if I'm honest, and I'm not sure if the functionality will continue, but if it helps people out for now, that's a good thing.

**Comment by jason178544** [comment]
thank you so much , worked for me :) . is there a way to import a .json file , so we can import certain rules? like this version has scripted https://github.com/ArunasFalcon/IntuneFirewallRulesMigration 
I tried changing yours a litttle to see if i could get working but couldnt. maybe there was something im missing, i was going to see if you had a quick soultion?

**Reply by ennnbeee** [reply]
Yeah there isn't a quick solution, and rearchitecting a Microsoft PowerShell script isn't the easiest 😅.

It's probably quicker allowing the script to run and create the rules, then just removing them from the policy in Intune, then exporting the Settings Catalog policy to JSON.
